Mesher_level/include/CGAL/Mesh_optimization_return_code.h

```diff
@@ -18,7 +18,10 @@
 #define CGAL_MESH_OPTIMIZATION_RETURN_CODE_H
 
 namespace CGAL {
-
+  /*!
+   * \brief bla bla
+   @todo  document correctly
```

Comment:
It is already documented in [Mesh_2](https://doc.cgal.org/latest/Mesh_2/group__PkgMesh2Enum.html#gab9fe60482a45120b3c061a8a4ec9018d) and the current doc of the return type of [lloyd_optimize_mesh_3()](https://doc.cgal.org/latest/Mesh_3/group__PkgMesh3Functions.html#gaa3a279f64022f2bc51935f946dd8a50e) points to Mesh_2

Reply:
`Mesh_2` as a dependency of `Mesh_3`?    It is in Mesher_level but that is probably a technicality for packaging/licensing questions.

Reply:
yes, to be accessible from both Mesh_2 and Mesh_3. However the doc is the same, and should be made accessible from Mesh_3.

Reply:
@sloriot any suggestion where to put cross-package error codes ?

Reply:
STL_extension?

Reply:
There are already files common to Mesh_2 and Mesh_3 in `Mesher_level/include/CGAL/Meshes/`. Maybe here.

Reply:
The question is   "In which manual chapter"

Reply:
Does it actually make sense to have the same enum for both, or is it done for convenience? In particular, don't you anyway want to document different things for the different values of the enum depending on the function? If that's the case, then maybe the enum should not have a doc at all, and all could be doc in each package. STL_extension is also an option for a shared doc, without or with a different name if you want to make it even more generic (but without an existing application I'd avoid that).

Reply:
Actually the enum is quite generic, has really the same meaning everywhere it is used, and in my opinion it would be a pity to duplicate it or its documentation.
It could probably be used in other functions too, like the PMP smoothing functions, if we gave them a time limit.

Reply:
Then it should be moved to STL extension (maybe removing mesh from the name?)